force-app/main/default/classes/ESA_cls_caseTriggerHandler.cls

```diff
@@ -289,4 +293,39 @@ public with sharing class ESA_cls_caseTriggerHandler {
             System.debug('Exception caught: ' + err.getMessage());
```

Comment:
remove debug log

Reply:
it is try and catch block, debug statement is used to capture the exception message.

Reply:
@yashrohan-cgi , we do not have a frame setup for selector class.